Farmland.py

```diff
@@ -1,205 +0,0 @@
-#This contains the two classes of BarrenLand and FertileLand
-
-import numpy.matlib
-import numpy as np
-
-#class BarrenLand holds information about the barren land,
-class BarrenLand:
-
-    #input is the string with four integers seperated by a space
-    def __init__(self, LowerLeftVertex=-1, UpperRightVertex=-1, stringVertex= '',fieldwidth = 400,fieldheight = 600):
-        if stringVertex != '':
-            # vertex list seperates the string into a list of 4 integers
-            self.vertexList = list(map(int, stringVertex.split()))
-        else :
-            self.vertexList = [LowerLeftVertex[0],LowerLeftVertex[1], UpperRightVertex[0], UpperRightVertex[1]]
-
-        if self.vertexList[0] < 0 or self.vertexList[1] < 0 or self.vertexList[2] >= fieldwidth or self.vertexList[3] >=fieldheight:
-            raise ValueError("The input of the vertex is out of bounds!")
-
-        #saves the vertex locations as attributes
-        self.lowerLeftX = self.vertexList[0]
-        self.lowerLeftY = self.vertexList[1]
-        self.upperRightX = self.vertexList[2]
-        self.upperRightY = self.vertexList[3]
-
-        #new attributes that describe the shape of the rectangle
-        self.height = 1+self.upperRightY - self.lowerLeftY
-        self.width = 1+self.upperRightX- self.lowerLeftX
-        self.area = self.height * self.width
-
-
-    #returns the individual squares of the barren land as points
-    def generatePoints(self):
-        Points = []
-
-        for i in range(0, self.width):
-            for j in range(0, self.height):
-                Points.append( [self.lowerLeftX + i, self.lowerLeftY + j])
-
-        return Points
-
-    #Prints BarrenLand coordinates
-    def getVertices(self):
-        LowerLeftVertex = [self.lowerLeftX, self.lowerLeftY]
-        UpperRightVertex = [self.upperRightX, self.upperRightY]
-
-        print("The lower left vertex is ", LowerLeftVertex )
-        print("The upper right vertex is ", UpperRightVertex)
-        return
-
-
-#This class is for the fertile land
-class FertileLand:
-
-    def __init__(self,fieldwidth= 400,fieldheight = 600):
-
-        #FertilePoints tells us if each 1 by 1 square is fertile or barren
-        #one represents fertile, zero represents barren
-        self.fertilePoints = np.matlib.ones((fieldwidth,fieldheight))
-        self.width = fieldwidth
-        self.height = fieldheight
-        self.totalArea = self.width * self.height #Holds the area of all fertile land
-
-        #seperated plots of land and number, we don't initialize this until removeBarren
-        self.plots_of_Land =[]  #0 if barren, >0 for the number of plot
-        self.number_of_Plots = 0 #counts the number of plots
-
-    #Prints FertilePoints
-    def printLand(self):
-        for i in range(self.height-1,0,-1):
-            for j in range (0,self.width):
-                print(self.fertilePoints[j,i], " ", end='')

```

Comment:
Have you tried running your code? When I run it, I get a syntax error for `end=''`. When I remove that, it runs fine.

Reply:
It works fine for me! Should I even keep that method? I only used it for helping myself write the code, I never call it